CHANGELOG.md

```diff
@@ -4,16 +4,22 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- CPU and memory limits are now configurable ([#347]).
+
+[#347]: https://github.com/stackabletech/opa-operator/pull/347
+
 ## [0.10.0] - 2022-09-06
 
 ### Changed
 
 - Include chart name when installing with a custom release name ([#313], [#314]).
 - `operator-rs` `0.15.0` -> `0.22.0` ([#315]).
 
-[#313]: https://github.com/stackabletech/trino-operator/pull/313
-[#314]: https://github.com/stackabletech/trino-operator/pull/314
-[#315]: https://github.com/stackabletech/trino-operator/pull/315
+[#313]: https://github.com/stackabletech/opa-operator/pull/313
+[#314]: https://github.com/stackabletech/opa-operator/pull/314
+[#315]: https://github.com/stackabletech/opa-operator/pull/315
```

Comment:
```suggestion
[#315]: https://github.com/stackabletech/opa-operator/pull/315
[#347]: https://github.com/stackabletech/opa-operator/pull/347
```

Reply:
I dont quite get this? This PR (347) is under unreleased.

Reply:
Oh I'm stupid, my fault